rewrite-java/src/main/java/org/openrewrite/java/style/UnnecessaryParenthesesStyle.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.style;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.With;
+import lombok.experimental.FieldDefaults;
+import org.openrewrite.java.JavaStyle;
+
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+@Data
+@With
+public class UnnecessaryParenthesesStyle implements JavaStyle {
+    ParenthesesTokens parenthesesTokens;
+
+    @FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+    @Data
+    @With
+    public static class ParenthesesTokens {
```

Comment:
you can drop the `Tokens` inner class altogether and just move these properties up to the style. "tokens" is just a checkstyle manifestation of this still, and doesn't need to occur here.